kitchen-tests/Berksfile.lock

```diff
@@ -3,7 +3,7 @@ DEPENDENCIES
     path: cookbooks/base
   php (~> 1.5.0)
   resolver
-    git: git://github.com/chef-cookbooks/resolver.git
+    git: https://github.com/chef-cookbooks/resolver.git
```

Comment:
yeah I think we always prefer https. Github started timing out on jenkins a few weeks back and switching to https fixed that